odl/set/domain.py

```diff
@@ -431,7 +432,7 @@ def measure(self, ndim=None):
         elif ndim > self.true_ndim:
             return 0.0
         else:
-            return np.prod((self.max_pt - self.min_pt)[self.inondeg])
+            return np.prod(self.extent()[self.nondegen_byaxis])
```

Comment:
`min` and `max` are methods in order to conform to the numpy interface, which allows these to be used with numpy in a duck-typing style. Otherwise we provide `min_pt` etc which do the same.

I'll propose a change.